### ISSUE TITLE
Customize drop area / preview

### ISSUE THREAD

**Issue by pbouttier** [hanshengchiu/reorderables#185]
Hi,

today the drop area looks like this 

`Widget spacing = _draggingWidget == null
          ? SizedBox.fromSize(size: _dropAreaSize)
          : Opacity(opacity: 0.2, child: _draggingWidget);`

We have a kind of preview of the element, would it be possible to customize it? For example, to simply have "nothing" (as if opacity = 0) and have it render like the official widget for reordering (I don't use it for other reasons)?

Thanks

**Comment by muxonn** [comment]
I totally agree with you, to be honest this feature should seem kinda obvious.
